packages/components/src/components/Alert/Alert.tsx

```diff
@@ -43,13 +43,19 @@ export type AlertProps = {
       header: string
       /** True if Alert should start expanded. Defaults to false */
       initializeExpanded?: boolean
+      /** Optional passthrough function for expand event */
+      onExpand?: () => void
```

Comment:
Not sure if we wanted to have an `analytics` object like with Link here. I think an object is a good idea for: consistency across components, setting it aside as non-functionality changing set of props, and more flexibility expanding analytics for a component in the future if any edge cases warrant it.

If doing that, probably fine to drop the `analytics` prop in the general list and not worry about locking down apps sending analytics that aren't possible depending on how they functionally configured it--as it's non-functional, I see no issue dropping them on the floor. It might even simplify how apps leverage it to just have a wrapper that always sends expand/collapse analytics even when they can't be hit.

Reply:
> Link sort of breaks this proposed pattern since it includes analytics.onPress though. We could potentially remove it if it's not already being used.

It is needed in Link due to us handling the onPress logic on Link variants where we can. Outside attactments/custom/calendar where they have to, they are unlikely to send custom onPress logic where they could do onPress analytics since doing so would overwrite the built in onPress behavior.